webapp/controller/ListReport.controller.js

```diff
@@ -64,17 +66,42 @@ sap.ui.define([
 			const oSearcherModel = new JSONModel({
 				field: ""
 			})
-			oModel.loadData("./model/products.json", "", false, "", true);
-			oModel.loadData("./model/categories.json", "", false, "", true);
-			oModel.loadData("./model/suppliers.json", "", false, "", true);
-			
-			this.getView().setModel(oModel);
+			const oDeleteButtonModel = new JSONModel({
+				Enable: false
+			})
+
+			this.getView().setModel(oDeleteButtonModel, "DeleteModel");
```

Comment:
As we discussed yesterday, create a single model called something like "ui" / "view" and store page's state in there